### ISSUE TITLE
Java.enumerateLoadedClasses produces Error: invalid arm system register

### ISSUE THREAD

**Issue by gorelics** [frida/frida-java#115]
Android 6.0 armeabi-v7a.  frida-server-12.6.5-android-arm. When trying `Java.enumerateLoadedClasses`  error occurred `Error: invalid arm system register`. Any suggestions?

**Comment by 0x6273** [comment]
I have the same issue on arm64 android 28.

```
-> Java.perform(function(){Java.enumerateLoadedClasses({"onMatch":function(className){ console.log(className) },"onComplete":function(){}})})
Error: invalid arm system register
    at frida/node_modules/frida-java/lib/android.js:1922
    at forEach (native)
    at Qe (frida/node_modules/frida-java/lib/android.js:1976)
    at frida/node_modules/frida-java/lib/android.js:1506
    at [anon] (native)
    at frida/runtime/core.js:199
    at Be (frida/node_modules/frida-java/lib/android.js:1510)
    at frida/node_modules/frida-java/lib/android.js:2261
    at ue (frida/node_modules/frida-java/lib/android.js:831)
    [...]
undefined
TypeError: cannot write property '_code' of null
    at [anon] (../../../frida-gum/bindings/gumjs/duktape.c:57666)
    at Be (frida/node_modules/frida-java/lib/android.js:1510)
    at frida/node_modules/frida-java/lib/android.js:2261
    at ue (frida/node_modules/frida-java/lib/android.js:831)
    at /_java.js:2677
    at frida/node_modules/frida-java/index.js:110
    at [anon] (input:1)
    at frida/node_modules/frida-java/lib/vm.js:42
    at E (frida/node_modules/frida-java/index.js:348)
    at frida/node_modules/frida-java/index.js:300
    at frida/node_modules/frida-java/lib/vm.js:42
    [...]```

**Comment by amankevich** [comment]
Got same issue. When I am calling Java.choose, it produces the error:

> Error: invalid arm system register
>     at frida/node_modules/frida-java/lib/android.js:1922
>     at forEach (native)
>     at Qe (frida/node_modules/frida-java/lib/android.js:1976)
>     at frida/node_modules/frida-java/lib/android.js:1506
>     at [anon] (native)
>     at frida/runtime/core.js:199
>     at Be (frida/node_modules/frida-java/lib/android.js:1510)
>     at frida/node_modules/frida-java/lib/android.js:2261
>     at ue (frida/node_modules/frida-java/lib/android.js:831)
>     [...]
> TypeError: cannot write property '_code' of null
>     at [anon] (../../../frida-gum/bindings/gumjs/duktape.c:57666)
>     at Be (frida/node_modules/frida-java/lib/android.js:1510)
>     at frida/node_modules/frida-java/lib/android.js:2261
>     at ue (frida/node_modules/frida-java/lib/android.js:831)
>     at /_java.js:4214
>     at frida/node_modules/frida-java/index.js:380
>     at /repl1.js:13
>     at frida/node_modules/frida-java/lib/vm.js:42
>     at E (frida/node_modules/frida-java/index.js:348)
>     at frida/node_modules/frida-java/index.js:300
>     at frida/node_modules/frida-java/lib/vm.js:42

Using `frida-server-12.6.5-android-arm` on Android 6.0.1

**Comment by gorelics** [comment]
Fixed in 12.6.6
